x/sequencer/types/messages.go

```diff
@@ -126,3 +128,32 @@ func (msg *MsgUnbond) GetSigners() []sdk.AccAddress {
 	}
 	return []sdk.AccAddress{creator}
 }
+
+/* ---------------------------- MsgIncreaseBond ---------------------------- */
+func NewMsgIncreaseBond(creator string, addAmount sdk.Coin) *MsgIncreaseBond {
+	return &MsgIncreaseBond{
+		Creator:   creator,
+		AddAmount: addAmount,
+	}
+}
+
+func (msg *MsgIncreaseBond) ValidateBasic() error {
+	_, err := sdk.AccAddressFromBech32(msg.Creator)
+	if err != nil {
+		return errorsmod.Wrapf(ErrInvalidAddress, "invalid creator address (%s)", err)
+	}
+
+	if !msg.AddAmount.IsValid() {
```

Comment:
this doesn't check if the added amount is `0`. So, it will be possible to send `0coin` when increasing.
We need to check that the amount is positive, and have a test case for it